docs/make.jl

```diff
@@ -21,6 +21,8 @@ makedocs(
     pages = Any[
         "Introduction" => "index.md",
         "First Steps with DataFrames.jl" => "man/basics.md",
+        "Joins" => "man/joins_new.md",
```

Comment:
this structure does not seem to be good.
Can you please make the "First Steps with DataFrames.jl" a top level tutorial page, and then have sections as separate pages in it (actually already https://dataframes.juliadata.org/stable/man/basics/ is probably too long and should be split into 2 separate pages; but this can be done later in a separate PR)